### PR TITLE
fix: add DO NOT EDIT comment

### LINKED ISSUE #58: Need DO NOT EDIT comment
Pulsar generated files are missing a comment like:
```
// Code generated by protoc-gen-go-pulsar. DO NOT EDIT.
```

### PR DESCRIPTION
**Pull request by aaronc**
closes #58

### CODE REVIEW

**Comment by aaronc** [comment]
I couldn't figure out how to generate the test .pulsar.go files. Could one of you give me a hint @fdymylja @technicallyty ?

**Comment by fdymylja** [comment]
> I couldn't figure out how to generate the test .pulsar.go files. Could one of you give me a hint @fdymylja @technicallyty ?

make pulsar

leftover before buf era

File: cmd/protoc-gen-go-pulsar/main.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ func generateAllFiles(plugin *protogen.Plugin, featureNames []string, poolable O
 		}
 
 		gf := plugin.NewGeneratedFile(file.GeneratedFilenamePrefix+".pulsar.go", file.GoImportPath)
+		gf.P("// Code generated by protoc-gen-go-pulsar. DO NOT EDIT.")
```
Comment (fdymylja) [inline cmd/protoc-gen-go-pulsar/main.go:75]: I had removed this because my IDE wouldn't process errors on generated files 🤣

**Review by fdymylja — APPROVED** [review]
pre-approving.

waiting on pulsar test files gen

**Comment by aaronc** [comment]
pushed pulsar test gen files

**Pull request merged**